cmd/restic/cmd_cat.go

```diff
@@ -143,7 +143,8 @@ func runCat(ctx context.Context, gopts GlobalOptions, args []string) error {
 		return err
 
 	case "blob":
-		err = repo.LoadIndex(ctx)
+		bar := newProgressMax(!gopts.Quiet, 0, "index files loaded")
```

Comment:
Missing `bar.Done()` call. This also applies to all files below. It would be an option to let `LoadIndex(ctx, bar)` automatically call `bar.Done()`